src/client/Microsoft.Identity.Client/Internal/Requests/SilentRequest.cs

```diff
@@ -75,9 +77,12 @@ private async Task<IAccount> GetAccountFromParamsOrLoginHintAsync(AcquireTokenSi
         internal async override Task PreRunAsync()
         {
 
-            if (ServiceBundle.PlatformProxy.CanBrokerSupportSilentAuth() && 
-                AuthenticationRequestParameters.IsBrokerConfigured)
+            if (ServiceBundle.PlatformProxy.CanBrokerSupportSilentAuth() &&
+                AuthenticationRequestParameters.IsBrokerConfigured &&
```

Comment:
nit: you can store this bool and then use it later, so you don't have to do all the checks again in `ExecuteAsync()`